apps/blockchain/test/blockchain_test.exs

```diff
@@ -18,8 +18,14 @@ defmodule BlockchainTest do
     "Homestead" => [],
     "TangerineWhistle" => [],
     "SpuriousDragon" => [],
-    "Byzantium" => String.split(@failing_byzantium_tests, "\n"),
-    "Constantinople" => String.split(@failing_constantinople_tests, "\n"),
+    "Byzantium" =>
+      @failing_byzantium_tests
+      |> String.split("\n")
+      |> Enum.reject(fn path -> path == "" || path == " " end),
```

Comment:
I feel like this could be cleaner. Could we update the script that writes this file not to write a newline instead? Or can we go back to listing the failing tests inline?

Reply:
@masonforest  good idea. We can rid of external files because not many failing tests left